src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/fabric/FabricMinecraftTransformer.kt

```diff
@@ -60,4 +68,84 @@ abstract class FabricMinecraftTransformer(
             "-Dfabric.remapClasspathFile=${intermediaryClasspath}"
         )
     }
+
+    override fun afterRemap(baseMinecraft: MinecraftJar): MinecraftJar {
+        val baseMinecraft = super.afterRemap(baseMinecraft)
+
+        val injections = hashMapOf<String, List<String>>()
+
+        this.collectInterfaceInjections(baseMinecraft, injections)
+
+        return if (injections.isNotEmpty()) {
+            val oldSuffix = if (baseMinecraft.awOrAt != null) baseMinecraft.awOrAt + "+" else ""
+
+            val output = MinecraftJar(
+                baseMinecraft,
+                parentPath = provider.localCache.resolve("fabric").createDirectories(),
+                awOrAt = "${oldSuffix}ii+${injections.hashCode()}"
+            );
+
+            if (!output.path.exists() || project.unimined.forceReload) {
+                if (InterfaceInjectionMinecraftTransformer.transform(
+                        injections,
+                        baseMinecraft.path,
+                        output.path,
+                        project.logger
+                    )
+                ) {
+                    output
+                } else baseMinecraft
+            } else output
+        } else baseMinecraft
+    }
+
+    private fun collectInterfaceInjections(baseMinecraft: MinecraftJar, injections: HashMap<String, List<String>>) {
+        val modJsonPath = this.getModJsonPath()
+
+        if (modJsonPath != null && modJsonPath.exists()) {
+            val json = JsonParser.parseReader(InputStreamReader(Files.newInputStream(modJsonPath.toPath()))).asJsonObject
+
+            val custom = json.getAsJsonObject("custom")
```

Comment:
Actually, [the location of this property appears to be different in QMJ format](https://github.com/QuiltMC/quilt-loom/blob/dev/1.4/patches/0003-Support-QMJ.patch#L1917), but you're treating all Mod JSONs the same.

Reply:
Currently it's only implemented for Fabric based projects, not Fabric like.
The reason why is because I didn't know how they are handled in Quilt.

Reply:
Implemented quilt support